udpa/annotations/status.proto

```diff
@@ -13,4 +13,10 @@ extend google.protobuf.FileOptions {
 message StatusAnnotation {
   // The entity is work-in-progress and subject to breaking changes.
   bool work_in_progress = 1;
+
+  // The entity represents the version at which active development for this
+  // package should take place. Earlier versions are considered frozen and not
+  // eligible for changes, later versions are typically machine generated from
+  // this active development version.
+  bool active = 2;
```

Comment:
How do you envision this being updated in the future? Will it be mostly automatic?

Reply:
Not sure TBH, I think it's not a huge pain at API clock cycle events (EOY, EOQ1) to have someone assemble a manual PR. In theory we could write some tooling to help here, but not sure if it would payoff.

Reply:
I'm on the fence whether this should be `active` or `frozen`, or should we make it enum? We will soon have v2 v3 and v4alpha and they are all in different status.

Reply:
Let's have an enum with `[FROZEN, ACTIVE, NEXT_MAJOR_VERSION_CANDIDATE]`?

Reply:
Updated, PTAL.